tensorrt_llm/_torch/attention_backend/sparse/kernel.py

```
@@ -1828,67 +1837,84 @@ def _convert_req_index_to_global_index_kernel_with_stride_factor(
-        NUM_TOPK_TOKENS: int = 2048,
-        BLOCK_N: int = 128,  # tile width along columns
-        stride_factor:
+    req_id: torch.Tensor,  # int32 [num_tokens]
+    block_table: torch.Tensor,  # int32 [num_requests, max_num_blocks_per_req]
+    token_indices: torch.
+    Tensor,  # int32 [num_kv_heads, num_tokens, NUM_TOPK_TOKENS]
+    BLOCK_SIZE: int,
+    NUM_TOPK_TOKENS: int = 2048,
+    BLOCK_N: int = 128,  # tile width along columns
+    stride_factor:
     int = None,  # for strided memory layout (with layer interleaving), defaults to BLOCK_SIZE
```

Ruff (0.14.13)

PEP 484 prohibits implicit `Optional`

Convert to `T | None`

(RUF013)